### PR TITLE
test: `changedCookieHeaderValue` and `expiredCookieHeaderValue`

### PR DESCRIPTION
**Pull request by johannes-lindgren**
Test for `changedCookieHeaderValue` and `expiredCookieHeaderValue`

These functions return the value of the Set-Cookie header for when you want to update the value (`changedCookieHeaderValue`) and when you want to expire the cookie (`expiredCookieHeaderValue`)

### CODE REVIEW

File: src/utils/cookie/expiredCookieHeaderValue/expiredCookieHeaderValue.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  it('has no value', () => {
+    expect(expiredCookieHeaderValue(cookieName)).toContain(`${cookieName}=""`)
+  })
+  it('is secure', () => {
+    expect(expiredCookieHeaderValue(cookieName)).toContain('; secure')
+  })
+  it("has the attribute 'samesite=none'", () => {
+    // TODO is this attribute really necessary?
+    expect(expiredCookieHeaderValue(cookieName)).toContain('; samesite=none')
+  })
```
Comment (eunjae-lee) [inline src/utils/cookie/expiredCookieHeaderValue/expiredCookieHeaderValue.test.ts:23]: I'm not sure about this, but what happens if you remove it?

File: src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    )
+  })
+  it.todo('url encodes the value')
+  it('is secure', () => {
+    expect(changedCookieHeaderValue(cookieName, cookieValue)).toContain(
+      '; secure',
+    )
+  })
+  it("has the attribute 'samesite=none'", () => {
+    // TODO is this attribute really necessary?
```
Comment (eunjae-lee) [inline src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts:24]:
```suggestion
```

File: src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    expect(changedCookieHeaderValue(cookieName, cookieValue)).not.toContain(
+      '; expires',
+    )
+  })
+  it('has a value', () => {
+    expect(changedCookieHeaderValue(cookieName, cookieValue)).toMatch(
+      new RegExp(`^${cookieName}=.+;`),
+    )
+  })
+  it.todo('url encodes the value')
```
Comment (eunjae-lee) [inline src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts:17]: is it a todo for this PR?

Comment (johannes-lindgren) [reply src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.test.ts:17]: I'd like to leave it for the future. The JWT tokens are already URL encoded, so this is not really needed for this library. But it's potentially an issue if the library evolves.

**Review by eunjae-lee — APPROVED** [review]

**Pull request merged**